builders/integrations/indexers/subquery.md

```diff
@@ -0,0 +1,160 @@
+
+For example, SubQuery can filter and query EVM logs and transactions in addition to Substrate data sources. SubQuery introduces more advanced filters than other indexers, allowing filtering of non-contract transactions, transaction senders, contracts and indexed log arguments, so developers can build a wide variety of projects that cater to their specific data needs.
+
+## Getting Started {: #getting-started }
+
+To get started, you'll need to [create a SubQuery project](https://doc.subquery.network/create/introduction/){target=blank}. Once you have your project set up, you can familiarize yourself with [how to connect to and index data from Moonbeam, Moonriver, and Moonbase Alpha](https://doc.subquery.network/create/moonbeam/){target=blank}.
+
+From within your SubQuery project, you can add the custom data source as a dependency by running the following [NPM](https://www.npmjs.com/){target=blank} or [Yarn](https://yarnpkg.com/){target=blank} command:
+
+=== "NPM"
+    ```
+    npm install @subql/contract-processors
+    ```
+
+=== "Yarn"
+    ```
+    yarn add @subql/contract-processors
+    ```
+
+The process to index Moonbeam data takes only two steps: [adding the Moonbeam custom data source](#adding-the-moonbeam-custom-data-source){target=blank} and then [indexing the Moonbeam data](#indexing-moonbeam-data){target=blank}.
+
+## Adding the Moonbeam Custom Data Source {: #adding-the-moonbeam-custom-data-source }
+
+SubQuery has created a data processor specifically made to work with Moonbeam’s implementation of [Frontier](https://github.com/paritytech/frontier){target=blank}. It allows you to reference specific ABI resources used by the processor to parse arguments and the smart contract address that the events are from or the call is made to.
+
+You can add the [custom data source](https://doc.subquery.network/create/moonbeam/#data-source-spec){target=blank} to your `project.yaml` manifest file:
+
+```yaml
+dataSources:
+  - kind: substrate/Moonbeam
+    processor:
+      file: './node_modules/@subql/contract-processors/dist/moonbeam.js'
+      options: {...}
+    assets: {...}
+```
+
+## Indexing Moonbeam Data {: #indexing-moonbeam-data}
+
+Next, you can add handlers for the custom data source to your code. Just like a normal SubQuery project, you use a mapping function to transform off chain data to the GraphQL entities that you define. The difference is that instead of a `SubstrateEvent` or `SubstrateExtrinsic`, your mapping function will receive a [`MoonbeamCall`](https://doc.subquery.network/create/moonbeam/#moonbeamcall){target=blank} or [`MoonbeamEvent`](https://doc.subquery.network/create/moonbeam/#moonbeamevent){target=blank} which are based on Ether's [TransactionResponse](https://docs.ethers.io/v5/api/providers/types/#providers-TransactionResponse){target=blank} or [Log](https://docs.ethers.io/v5/api/providers/types/#providers-Log){target=blank} type.
```

Comment:
Similar to the comment above, if an Ethereum developer wants to use Subquery I would be  a little bit more specific